src/cascadia/TerminalControl/ControlCore.cpp

```diff
@@ -194,7 +195,7 @@ namespace winrt::Microsoft::Terminal::Control::implementation
             [weakTerminal = std::weak_ptr{ _terminal }]() {
                 if (const auto t = weakTerminal.lock())
                 {
-                    auto lock = t->LockForWriting();
+                    const auto lock = t->LockForWriting();

```

Comment:
I "normalized" all the various ways we named the lock variables. They're all `const auto` and named `lock` (instead of `guard`) now, because that was the most common denominator.